### PR TITLE
Add KSQL Aggregation / SUM recipe

### PR DESCRIPTION
**Pull request by rmoff**

### CODE REVIEW

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/continuous-query.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Since the output looks right, the next step is to make the query continuous. Issue the following to create a new table that is continuously populated by its query:
```
Comment (MichaelDrogalis) [inline _includes/tutorials/aggregating-sum/ksql/markup/dev/continuous-query.adoc]: Could use a bit of an explainer as to why we use a table here. Links would be great too.

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/print-output-topic.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+To check that it's working, print out the contents of the output stream's underlying topic:
```
Comment (MichaelDrogalis) [inline _includes/tutorials/aggregating-sum/ksql/markup/dev/print-output-topic.adoc]: One other thing worth mentioning here is how the `AS` keyword functions to make a new column, and thus a new named attribute on the map that's printed. Seems basic, but it looks me a bit to figure that out when I was learning KSQL.

**Comment by MichaelDrogalis** [comment]
It's also probably worth adjusting the permalink to something like `sum-a-stream-of-events` since that matches natural language a bit better.

File: _includes/tutorials/aggregating-sum/ksql/code/tutorial-steps/dev/transient-query.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+SELECT TITLE, 
+       SUM(TICKET_TOTAL_VALUE) AS TOTAL_VALUE
+FROM MOVIE_TICKET_SALES
+GROUP BY TITLE
```
Comment (MichaelDrogalis) [inline _includes/tutorials/aggregating-sum/ksql/code/tutorial-steps/dev/transient-query.sql]: One other thing to briefly touch on in the narrative is the function that group by plays with getting the right results.

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/create-movie-ticket-sales.adoc [hunk 4]
```diff
@@ -0,0 +1,7 @@
+First, you'll need to create a Kafka topic and stream to represent the ticket sales. 
+The statement below creates both at the same time. This stream contains the name of the movie and the price of the ticket to watch it. We model the price as an integer to make the example simple.
+Another important characteristic of the data is the timestamp column, `sale_ts`, which can be used for time-based windowing. KSQL can use either the Kafka message timestamp or a field from the message value as the timestamp. In this example, we'll use the latter, which is also known as _event time_.
```
Comment (MichaelDrogalis) [inline _includes/tutorials/aggregating-sum/ksql/markup/dev/create-movie-ticket-sales.adoc]: One other thing I noticed is that this tutorial goes out of its way to explain event time, but doesn't use it. Did you intend to use it in the queries? Worth leaving out if not.

File: _includes/tutorials/aggregating-sum/ksql/code/tutorial-steps/dev/populate-movie-ticket-sales.sql [hunk 5]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO MOVIE_TICKET_SALES (title, sale_ts, ticket_total_value) VALUES ('Die Hard','2019-07-18T10:00:00Z', 12);
```
Comment (MichaelDrogalis) [inline _includes/tutorials/aggregating-sum/ksql/code/tutorial-steps/dev/populate-movie-ticket-sales.sql]: Spacing between column values for title and timestamp.

**Comment by MichaelDrogalis** [comment]
I had one other thought: we might want to flip on EOS for this. We should note EOS usage, whether on or off, no matter what.

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/create-movie-ticket-sales.adoc (hunk 4 above)
Comment (tlberglund) [reply _includes/tutorials/aggregating-sum/ksql/markup/dev/create-movie-ticket-sales.adoc]: Good point. There is no window, so no real use for event time. Might as well just nuke it and the explanation, I think.

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/set-properties_buffer.adoc [hunk 6]
```diff
@@ -0,0 +1,5 @@
+For the purposes of this example only, we are also going to configure KSQL to buffer the aggregates as it builds them. This makes the query feel like it responds more slowly, but means that you get just one row per movie from which it is simpler to understand the concept:
```
[the remark this answers is not in the document]
Comment (tlberglund) [reply _includes/tutorials/aggregating-sum/ksql/markup/dev/set-properties_buffer.adoc]: I trust @rmoff that he had good reasons, but I don't know what they are. If you (@MichaelDrogalis) know, drop me some rough text in a comment and I'll incorporate it into the narrative.

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/print-output-topic.adoc (hunk 2 above)
Comment (tlberglund) [reply _includes/tutorials/aggregating-sum/ksql/markup/dev/print-output-topic.adoc]: Worked that into the previous step.

File: _includes/tutorials/aggregating-sum/ksql/code/tutorial-steps/dev/transient-query.sql (hunk 3 above)
Comment (tlberglund) [reply _includes/tutorials/aggregating-sum/ksql/code/tutorial-steps/dev/transient-query.sql]: I think I addressed that adequately in the next step. Your call.

**Pull request merged**